### ISSUE TITLE
whether the TDOA2 mode supports remote Settings of the base station

### ISSUE THREAD

**Issue by cqutyoulang** [bitcraze/lps-node-firmware#61]
I can do it witch the tdoa3 model, but the tdoa2 model doesn't work. But I saw the LPP data section in the tdoa2 source code.

**Comment by krichardsson** [comment]
Yes, TDoA2 also supports setting anchor positions from the Crazyflie (same functionality as TDoA3)

**Comment by knmcguire** [comment]
I'm transferring this issue to disscussions, as this is a question
